File: programmers/Lv_1/JeongJunKwon/같은 숫자는 싫어.java
import java.util.*;

public class Solution {
	public int[] solution(int []arr) {
        List<Integer> list = new ArrayList<>();
        int index = -1;
        for(int i = 0; i < arr.length; i++) {
            if(arr[i] != index) {
                list.add(arr[i]);
                index = arr[i];
            }
        }
        
        int[] answer = new int[list.size()];
        for(int i = 0; i < answer.length; i++) {
            answer[i] = list.get(i);
        }

        return answer;
	}
}

File: programmers/Lv_1/KimHyunseo/같은숫자는싫어.cpp
#include <vector>
#include <iostream>


using namespace std;

vector<int> solution(vector<int> arr)
{
	
	vector<int> answer;
	for (int i = 0; i < arr.size();)
	{
		answer.push_back(arr[i]);
		for (int j = i + 1; i < arr.size();j++)
		{
			if (arr[i] != arr[j]) {
				i = j;
				break;
			}
		}
	}
	return answer;
}

int main()
{
	vector<int> array = { 1,1,3,3,0,1,1 };
	solution(array);

}